open-sphere-base/mantle/src/main/java/io/opensphere/mantle/icon/impl/gui/IconChooserPanel.java

```diff
@@ -149,6 +149,12 @@
     /** The optional selected icon URL. */
     private String mySelectedUrl;
 
+    /** The icon display width */
+	public int myTileWidth;

```

Comment:
Make sure you've followed the steps here: https://github.com/ngageoint/opensphere-desktop/wiki/Coding-Standards to setup eclipse formatting, save actions, etc.  Then format each of these classes.